lib/locale/pl/strings.php

```diff
@@ -3603,4 +3603,6 @@
 $_LANG['vacation'] = 'urlop';
 $_LANG['duty'] = 'dyżur';
 
+$_LANG['If isset "true" when logout confirmation is required. Default "false"'] = 'Ustawienie opcji na "true" wymusza potwierdzenie wylogowania. Domyślnie wyłączone';
```

Comment:
Prawie idealnie tylko poproszę jeszcze o zmianę tego angielskiego tekstu na:
If set to "true" then logout confirmation is required. Default "false"